en_US/data-integration/rules.md

```diff
@@ -30,72 +52,98 @@ WHERE
+
+## Typical Use Cases
+
+### Action Monitoring
+
+Rules can be used to monitor the status of clients connected to the broker. 
+
+For example, if a client representing a door in a smart home is disconnected, a rule can be used to send a notification to a dashboard for the smart home. This helps ensure that important events are promptly acted upon.
+
+### Data Filtering
+
+In certain scenarios, only specific types of data are relevant, and processing all incoming data may not be necessary. 
+
+For example, in truck fleet management, vehicle sensors collect and report a large amount of operation data. However, the application platform may only be interested in data when the vehicle speed is greater than 40 km/h. 
+
+In this case, rules can be used to filter messages conditionally and only write data that meets the conditions to a message queue. This can help optimize system performance and reduce storage requirements.
+
+### Message Routing
+
+Rules can also be used to route messages to different topics based on the content of the message. This allows for more fine-grained control over message distribution. 
+
+For example, a rule can be used to route messages containing a temperature greater than 100 degrees to a topic for high-temperature alarms. This can help ensure that messages are delivered to the appropriate destination for further processing.
 
-- Action monitoring: in the development of smart home smart door locks, the door locks will be offline due to network, power failure, man-made damage and other reasons, resulting in abnormal functions. Use rule configuration to monitor offline events and push the fault information to the application service, so as to realize the ability of fault detection at the access layer at the first time;
+### Message Encoding and Decoding
 
-- Data filtering: in the truck fleet management of the Internet of vehicles, vehicle sensors collect and report a large amount of operation data. The application platform only focuses on the data when the vehicle speed is greater than 40 km/h. In this scenario, rules can be used to filter messages conditionally and write data that meets the conditions to the business message queue;
+For cases where the format transform of messages is needed, rules can be used to encode and decode messages as required.
 
-- Message routing: in the intelligent billing application, the terminal equipment distinguishes the service types through different topics. The charging service messages can be connected to the charging message queue through the configuration rules and send a confirmation notice to the service system after the messages arrive at the device end. The non charging information is connected to other message queues to realize the service message routing configuration;
+For example, a rule can be used to decode a message containing binary data into a JSON format, and then forward the JSON format message to the application platform for further processing. 
```

Comment:
", and then forward the JSON format message to the application platform for further processing." <- this can be removed